### ISSUE TITLE
Store initialization does not work in Safari

### ISSUE THREAD

**Issue by Gaulomatic** [mrpmorris/blazor-fluxor#82]
I have problems with the injected JavaScript which initializes the store on Apple devices. I am not exactly sure what causes it, but in Safari (on both macOS and iOS) the JS is not invoked. I am currently using a workaround - basically call the methods on my own - but it's not ideal. And really ugly.

```javascript
const InitializeStore = () =>
{
	if (!window.DotNet)
	{
		setTimeout(() => InitializeStore(), 0);
	}
	else
	{
		setTimeout(() => PerformCallback(), 0);
	}
};

const PerformCallback = () =>
{
	try
	{
		DotNet.invokeMethodAsync("Blazor.Fluxor", "Blazor.Fluxor.OnPageLoaded")
			.then(() => console.info("Store initialized"))
			.catch(() =>
			{
				setTimeout(() => PerformCallback(), 0)
			});
	}
	catch (ex)
	{
		setTimeout(() => PerformCallback(), 0);
	}
};

InitializeStore();
```

Any idea why Safari does not play nice?

**Comment by mrpmorris** [comment]
Can you get hold of me on Gitter so we can work this out together?

https://gitter.im/aspnet/Blazor

**Comment by mrpmorris** [comment]
Fixed in 1.1
